.changeset/weak-crews-judge.md

```diff
@@ -69,10 +71,10 @@ To change all instances of `theme.typography.lineHeights` to the new scale autom
 
 **Step2:** Run the codemod with the following command
 
-> use `--parser=tsx` for typescript projects and `--parser=jsx` for js projects
+> Checkout [jscodeshift docs](https://github.com/facebook/jscodeshift) for further cli usage help.
 
 ```sh
-npx migrate-typography ./YOUR_DIR --parser=tsx
+npx jscodeshift ./YOUR_DIR --extensions=tsx,ts,jsx,js -t ./node_modules/@razorpay/blade/codemods/migrate-typography/transformers/migrate-typography.ts --ignore-pattern="**/node_modules/**"
```

Comment:
why moved to this? the node_modules path might differ in monorepos

Reply:
Slack thread: https://razorpay.slack.com/archives/G01B3LQ9H0W/p1681108611394679?thread_ts=1681102804.037129&cid=G01B3LQ9H0W

In monorepos even with the approach before you'll need to CD into the monorepo package and then run the script since internally cli also just appended the node_module path.